src/docs/operator-guides/migrating-from-outside-sharetribe/index.md

```diff
@@ -28,33 +32,102 @@ Intermediary is an edn (https://github.com/edn-format/edn) data format
 that allows defining data rows for marketplaces. It supports creating
 links between rows via references.
 
+We recommend that you use an .edn library to generate the data and
+validate the .edn syntax to ensure that the file is up to standard.
+
 ### Format
 
 Intermediary is specified as
 [a map](https://clojure.org/reference/data_structures#Maps) with the
 following keys:
 
-- :ident - identifies the target marketplace by marketplace ident,
+- :ident - identifies the target marketplace by marketplace ID,
```

Comment:
Like `map` on row 41, should `seq` and `2-tuple` link to a Clojure reference for developers not familiar with Clojure data structures?

Reply:
Good point. 2-tuple is somewhat a generic programming concept but `seq` is Clojure-specific. I think we can use the N-tuple wording in the document but when first mentioning it, we could write it out:
> 2-tuple (an array with 2 entries)